src/db/migrations/20190403212306-create-article.js

```diff
@@ -0,0 +1,74 @@
+module.exports = {
```

Comment:
let's use `export default` instead of `module.exports` for consistency with es6. Please update your code below where this is necessary.

Reply:
Okay. Will do just that